documentation/docs/developers/connectors/task3.md

```diff
@@ -393,9 +419,32 @@ Returns an order amount step, a minimum amount increment for a given trading pai
 | --------------- | --------- | ----------------------------------------------------------- |
 | `min_increment` | `Decimal` | Minimum base asset size increment of specified trading pair |
 
-### (5) Additional Function(s)
+### (5) Managing Funding Information (Derivative)
 
-The list below contains the utility functions and descriptions that the `Exchange` class use.
+In addition to the account positions mentioned in [section 3](#3-tracking-orders-balances--positions), the
+`Derivative` class also keeps track of the funding payments and the relevant information pertaining to them.
+The class must detect when funding payments are issued and trigger [`FundingPaymentCompletedEvent`](https://github.com/CoinAlpha/hummingbot/blob/master/hummingbot/core/event/events.py#L222-L228)
+as necessary. The details of how this is achieved depend heavily on the given exchange's API. For instance, the 
+[Binance Perpetual derivative class](https://github.com/CoinAlpha/hummingbot/blob/master/hummingbot/connector/derivative/binance_perpetual/binance_perpetual_derivative.py)
+implements a separate polling loop that uses a REST API endpoint to request the information at the appropriate time (see `_funding_fee_polling_loop`), whereas the
+[dydx Perptual derivative class](https://github.com/CoinAlpha/hummingbot/blob/master/hummingbot/connector/derivative/dydx_perpetual/dydx_perpetual_derivative.py)
+receives the information from the websocket stream in the `_user_stream_event_listener()` method.
+
+The `_funding_payment_span` list contains two integers denoting the number of seconds before and after the funding
+period when active positions are considered by the exchange as being eligible for funding payment. If the exchange
+takes a single snapshot of the opened positions as opposed to a window, those values may be left to their defaults of
+zero.
+
+Finally, the `_funding_info` dictionary must be maintained. Much like the funding payments information, this is 
```

Comment:
Maybe it will be useful to detail the `FundingInfo` object and what are the key information we intend on capturing for each trading pair.